linux_os/guide/system/accounts/accounts-pam/password_quality/password_quality_pwquality/accounts_password_pam_ucredit/rule.yml

```diff
@@ -1,6 +1,6 @@
 documentation_complete: true
 
-prodtype: alinux2,fedora,ol7,ol8,ol9,rhel7,rhel8,rhel9,rhv4,sle12,sle15,ubuntu2004,ubuntu2204
+prodtype: alinux2,fedora,ol7,ol8,ol9,rhel7,rhel8,rhel9,rhv4,ubuntu2004,ubuntu2204
```

Comment:
@teacup-on-rockingchair , it is not recommended to remove `CCEs` from existing rules.
Please, take a look in the information about `identifiers` in the `rule` section of the documentation:
https://complianceascode.readthedocs.io/en/latest/manual/developer/06_contributing_with_content.html#rules

It may cause issues for users who use these rules, in tailored files, for example.

Reply:
understood :+1:  will revert those changes

Reply:
My point was that those were relatively easy changes and those rules don't behave OK with SLE anyways because of missing packages, but I guess that is something that is irreversible now :)